dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -596,9 +596,32 @@ public Tool registerManual(@ApiParam(hidden = true) @Parameter(hidden = true, na
             tool.setGitUrl(convertHttpsToSsh(tool.getGitUrl()));
         }
 
+        // Can't set tool license information here, far too many tests register a tool without a GitHub token
```

Comment:
add TODO for highlighting 
https://www.jetbrains.com/help/idea/using-todo.html

alternatively can `setToolLicenseInformation` skip tools without a github token to avoid it being dead code?